lib/Sub/Quote.pm

```diff
@@ -37,6 +35,41 @@ BEGIN {
   *_FLOAT_PRECISION = sub(){$precision};
 }
 
+BEGIN {
+  local $@;
+  if (eval { require XString; XString->VERSION(0.003) }) {
```

Comment:
We want it to be 0.005 to support 5.8

Reply:
That shouldn't matter. A user on 5.8 won't be able to install 0.003, so they won't ever be able to get to this check.

Reply:
Or more clearly, a 5.8 user would need to have installed 0.005 for the require to succeed, so the version check would already be passing.